app/src/main/kotlin/com/simplemobiletools/keyboard/views/MyKeyboardView.kt

```diff
@@ -1056,7 +1025,9 @@ class MyKeyboardView @JvmOverloads constructor(context: Context, attrs: Attribut
                     )
                     mMiniKeyboardCache[popupKey] = mMiniKeyboardContainer
                 } else {
-                    mMiniKeyboard = keyboardPopupBinding!!.miniKeyboardView
+                    //Using 'miniKeyboardView' from cache
+                    mMiniKeyboard = mMiniKeyboardCache[popupKey]?.let(KeyboardPopupKeyboardBinding::bind)?.miniKeyboardView
```

Comment:
simplify this line please, theres way too many things going on. Also avoid adding useless comments.